dune

```diff
@@ -1,6 +1,6 @@
 (env
  (dev
   (flags
-   (:standard -w +A-42-44-45-48-66 -warn-error +A-3))))
+   (:standard -w +A-42-44-45-48-66-70 -warn-error +A-3))))
```

Comment:
What is warning 70?

Reply:
"missing interface" -> which is sometimes wanted ;-)